integration_tests/src/main/python/date_time_test.py

```diff
@@ -671,3 +671,14 @@ def test_timestamp_millis_long_overflow():
 def test_timestamp_micros(data_gen):
     assert_gpu_and_cpu_are_equal_collect(
         lambda spark : unary_op_df(spark, data_gen).selectExpr("timestamp_micros(a)"))
+
+
+@pytest.mark.parametrize('parser_policy', ["LEGACY", "CORRECTED", "EXCEPTION"], ids=idfn)
```

Comment:
Any reason to use different quote marks? 
```suggestion
@pytest.mark.parametrize('parser_policy', ['LEGACY', 'CORRECTED', 'EXCEPTION'], ids=idfn)
```

Reply:
Done